### PR TITLE
Update options.md

### PR DESCRIPTION
**Pull request by jienagu**
I edited from the beginning to Text Results `collapse`. I will work on the rest later.

cf https://github.com/yihui/knitr/issues/1481

### CODE REVIEW

**Review by yihui — COMMENTED** [review]
Great! Do you want me to review the PR now, or wait until you are done with this page?

**Comment by jienagu** [comment]
How about review it from now so we don't have a backlog.

File: content/knitr/options.md
Comment (yihui) [inline content/knitr/options.md]: It is no longer necessary to mention Sweave here: knitr was created to replace Sweave, and I guess it has done its job long time ago :)

File: content/knitr/options.md
Comment (yihui) [inline content/knitr/options.md]: The example below is not Rnw, but Rmd.

```suggestion
Take Rmd files as an example. Usually we write chunk options in the form
```

File: content/knitr/options.md
Comment (yihui) [inline content/knitr/options.md]: I feel `in` is more appropriate here (or `inside`).

File: content/knitr/options.md
Comment (yihui) [inline content/knitr/options.md]: I prefer short sentences.

```suggestion
1. Chunk options must be written on one line. No line breaks are allowed
```

File: content/knitr/options.md
Comment (yihui) [inline content/knitr/options.md]: Actually this is true only for Rnw documents, so I'd rather not mention the substitution at all.

File: content/knitr/options.md
Comment (yihui) [inline content/knitr/options.md]:
```suggestion
  - In theory, the chunk label should be quoted as well. However, for the sake of
```

File: content/knitr/options.md
Comment (yihui) [inline content/knitr/options.md]: Let's just get rid of this bullet about Sweave.

**Review by yihui — CHANGES_REQUESTED** [review]
Good start! One major problem of the **knitr** documentation site is that it was created with Rnw/LaTeX documents in mind in the beginning. I'd like to mention Rnw less frequently, and use R Markdown whenever possible. That said, you don't have to do this at the moment. We can leave it to the next step after this PR is merged.

**Review by yihui — APPROVED** [review]
I just finished the rest of work by myself. Thanks again!

**Pull request merged**
